src/tests/ftest/erasurecode/rebuild_fio.yaml

```diff
@@ -41,6 +41,7 @@ server_config:
       scm_list: ["/dev/pmem0"]
       scm_mount: /mnt/daos0
       log_mask: ERR
+      targets: 2
```

Comment:
Any reason for changing the targets to 2? I would suggest to keep it maximum which is 8 unless I miss something.

Reply:
the reason is to workaround a placement problem - shard collocated problem as what we discussed in the EC meeting this week. that is only in special cornel case but could be hit in this test, I'll handle it in another separate patch.